pandera/errors.py

```diff
@@ -166,6 +166,8 @@ def _message(self, error_counts, schema_errors):
             msg += f"- {k}: {v}\n"
 
         def agg_failure_cases(df):
+            # FIXME(arne): hack to support unhashable types, wait for #260
```

Comment:
let's add the full url to the issue here: `https://github.com/unionai-oss/pandera/issues/260`